config.toml

```diff
@@ -241,7 +148,7 @@ weight = 10
 [[menu.community]]
 parent = "contribute"
 name = "GitHub"
-url = "https://github.com/ipfs/ipfs"
+url = "https://github.com/libp2p/libp2p"
 weight = 5
 [[menu.community]]
 parent = "contribute"
```

Comment:
I can't comment on an unchanged line, but note that 1 and 7 lines below here are two 404 links:

```
url = "https://github.com/ipfs/community/blob/master/contributing.md"
```
and
```
url = "https://github.com/ipfs/community/blob/master/CONTRIBUTING-2.md"
```